Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/equate/RemoveEquateCmd.java

```diff
@@ -58,7 +58,7 @@ public String getName() {
 	}
 
 	/**
-	 * @see ghidra.framework.cmd.Command#applyTo(ghidra.framework.plugintool.PluginTool, ghidra.framework.model.DomainObject)
+	 * @see ghidra.framework.cmd.Command#applyTo(ghidra.framework.model.DomainObject)
 	 */
```

Comment:
This can be removed

Reply:
Yes regarding you previous comment it probably could, but this change makes it at least "more correct then before" :D